src/assets/style/mapping.css

```diff
@@ -0,0 +1,18 @@
+smoothly-next-table {
```

Comment:
Defining it in :root doesn't work since then the variables will "resolve" long before any color changes applies to --smoothly-color an thus will not take effect.